### DISCUSSION TITLE
How to change the default style for a new node

### DISCUSSION THREAD

**Discussion by theelectricsoul** [freeplane/freeplane#1253] [Help] [answered]
Hello :)

**Scenario**
I insert a new child node.

**Behaviour**
Freeplane inserts a node and gives it the style 'Default'

**Needed behaviour**
Freeplane inserts a node and gives it the style 'Level X', according to the current level.

**Question**
Which setting did I mess up :)

**Comment by macmarrum** [comment]
Try `Format->Automatic level styles->for all nodes` and `Format->Style for new nodes->Assign no style`

**Reply by JulianSMoore** [reply]
Having just spent quite some time trying to figure out why my automatic level styling wasn't being applied, may I ask what the use-case for conflicting/overriding style settings on creation is? (Assuming it's a deliberate choice.)

I found it very confusing because it seems that Style For New Nodes overrides Automatic Level Styling, which means I would have expected ALS to be unavailable unless SFNN was set to Assign No Style.

(Although "Default" is also perhaps a possibility here, because ALS is surely a "default" for based on the level number.)

What does this separation of style on creation possibilities allow that would otherwise not have been possible?

**Reply by macmarrum** [reply]
Please see https://docs.freeplane.org/user-documentation/styles.html, especially about the concept of layered Styles.

**Comment by theelectricsoul** [comment, marked as answer]
# Solution
![image](https://github.com/freeplane/freeplane/assets/10260832/47468d31-2171-4773-b7a2-8d0f546c4512)

@macmarrum
